### ISSUE TITLE
Error: probably not a valid book

### ISSUE THREAD

**Issue by michalastocki** [alexgand/springer_free_books#113]
Hello, do you have any ideas why it's not downloading any of the books? I have all the dependencies and the python code seems to be running, but ot doesn't download anything right now, skipping every book.

Does it mean that Springer recovered all the books from their site?

Thanks!

![image](https://user-images.githubusercontent.com/51298466/82101395-af9e2480-970c-11ea-93eb-e051360ac8bb.png)

**Comment by wallacefsilva** [comment]
I'm having the same issues. It seems they implemented a security check via recaptcha...

**Comment by renanxcortes** [comment]
Yep.. it seems I'm facing the same type of issue in the R version of this package (https://github.com/renanxcortes/springerQuarantineBooksR/issues/53). Any clue on how to solve this issue?

**Comment by lgabs** [comment]
I'm getting the same errors:

![image](https://user-images.githubusercontent.com/27077298/82760240-aece5800-9dc8-11ea-93bd-a539c6e5b959.png)

**Comment by renanxcortes** [comment]
It seems like springer tweaked the reCaptcha step and the recent workaround is not enough to avoid getting flawed files :(

**Comment by chaosAD** [comment]
I have a raw working prototype to get around the latest hurdle. My code also manipulates the HTTP `cookies` to get past it, but in an elaborate way. I see Springer put other cookies that are used to track downloading users as well. This makes it easier for their monitoring software to analyze and then flag a downloading bot. Obviously, they do not allow downloading using bots and I feel we are playing cat and mouse with Springer. For that reason, I am not going to pursue on this.

**Comment by SanJJ1** [comment]
So does this mean that this project is dead?

**Comment by pbl987** [comment]
> 
> 
> I have a raw working prototype to get around the latest hurdle. My code also manipulates the HTTP `cookies` to get past it, but in an elaborate way. I see Springer put other cookies that are used to track downloading users as well. This makes it easier for their monitoring software to analyze and then flag a downloading bot. Obviously, they do not allow downloading using bots and I feel we are playing cat and mouse with Springer. For that reason, I am not going to pursue on this.

There is no need for playing cat and mouse - JDownloader does that!
They can solve automatically weak captchas, and semi-automatically with google.

I want to save at least the covid-package, **as this wont be available in two weeks!!!**
Currently 389 books are available, it would be doable to enter a captcha every 5 download or so.

Could you please reconsider helping the project?
